clients/cli/src/unwrap.rs

```diff
@@ -35,10 +37,6 @@ pub struct UnwrapArgs {
     #[clap(value_parser = parse_pubkey)]
     pub wrapped_token_account: Pubkey,
 
-    /// The address of the escrow account holding the unwrapped tokens
-    #[clap(value_parser = parse_pubkey)]
-    pub escrow_account: Pubkey,
-
     /// The address of the token account to receive the unwrapped tokens
     #[clap(value_parser = parse_pubkey)]
     pub unwrapped_token_recipient: Pubkey,
```

Comment:
This can be done in a separate command, but it would be good to have a way to find out: "Given a mint address, is this a wrapped token mint? If so, what's it wrapping?"